lib/ansible/modules/web_infrastructure/jenkins_script.py

```diff
@@ -101,6 +108,22 @@
     password: admin
     url: https://localhost
     validate_certs: no
+#
+# Example of how to use the params
+#
+# Define a variable and specify all default parameters you want to use across
+# all jenkins_plugin calls:
+#
+# my_jenkins_params:
```

Comment:
This is confusing ... if the example is meant to show creating a variable to be used by the params option then don't comment out that variable.

Reply:
This was a direct copy from the way it was documented in jenkins_plugin.py:

https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/web_infrastructure/jenkins_plugin.py#L173

Reply:
There was a recent fix to `jenkins_plugin` to avoid exposing the `url_password` in logs, perhaps it is worth bringing that idea to this plugin as well. Here is the change:

https://github.com/ansible/ansible/commit/863fcb5aced55fd9ce3080905bd6405c8bfd0511#diff-d2e810bf269df1e341b77e8baae7d35e